xml/System.Net.Sockets/Socket.xml

```diff
@@ -4181,7 +4173,7 @@ This method populates the <xref:System.Net.Sockets.Socket> instance with data ga
   
  ]]></format>
         </remarks>
-        <exception cref="T:System.NotSupportedException">This property can be set only for sockets in the <see cref="F:System.Net.Sockets.AddressFamily.InterNetwork" /> or <see cref="F:System.Net.Sockets.AddressFamily.InterNetworkV6" /> families.</exception>
+        <exception cref="T:System.NotSupportedException">The socket is not in the <see cref="F:System.Net.Sockets.AddressFamily.InterNetwork" /> family.</exception>
```

Comment:
`InterNetworkV6` should be added back:

```suggestion
        The socket is not in the <see cref="F:System.Net.Sockets.AddressFamily.InterNetwork" /> or <see cref="F:System.Net.Sockets.AddressFamily.InterNetworkV6" /> families.
```

Reply:
@antonfirsov I removed it based on [this line](https://github.com/dotnet/runtime/blob/8b6f3cebcb564d4623f16fe4906f344880a9a693/src/libraries/System.Net.Sockets/src/System/Net/Sockets/Socket.cs#L657), but I'll add it back.

Reply:
Hm, my mistake, I haven't checked the code, and I'm not familiar with the option actually.

@wfurt @tmds is `IP_DONTFRAGMENT` IPV4-only, or do we have a bug on the [mentioned line](https://github.com/dotnet/runtime/blob/8b6f3cebcb564d4623f16fe4906f344880a9a693/src/libraries/System.Net.Sockets/src/System/Net/Sockets/Socket.cs#L657)?

Reply:
There is no fragmentation in IPv6. Replaced with mandatory PMTU discovery.

Reply:
@gewarren I was wrong sorry again, you can revert if you want.

Reply:
No worries @antonfirsov. Does this line in the remarks section need to be updated at all?

```
Datagrams may be fragmented by the sending host (all Internet Protocol versions) or an intermediate router (Internet Protocol Version 4 only).
```